sonic-xcvrd/xcvrd/xcvrd_utilities/port_mapping.py

```diff
@@ -1,19 +1,25 @@
+import threading
 from sonic_py_common import daemon_base
 from sonic_py_common import multi_asic
 from sonic_py_common.interface import backplane_prefix, inband_prefix, recirc_prefix
 from swsscommon import swsscommon
 
 SELECT_TIMEOUT_MSECS = 1000
+DEFAULT_PORT_TBL_MAP = [
```

Comment:
@longhuan-cisco Thanks this looks much better than previous proposed solution. May I have one more suggestion. Currently this map is fixed to static filtering if the subscriber threads could pass this filter object while subscribing, then this will make the solution more generic like multiple threads could subscribe to different combination of events: DB/Table/Fields

Reply:
The code had already been made generic by allowing custom port_tbl_map argument (DB/table/fields)

https://github.com/sonic-net/sonic-platform-daemons/blob/abcf4c425a81befdce316317cd0c74e46a10bc18/sonic-xcvrd/xcvrd/xcvrd_utilities/port_event_helper.py#L48-L51